lte/cloud/go/services/cellular/protos/cellular_service.proto

```diff
@@ -137,6 +137,7 @@ message NetworkEPCConfig {
     }
     map<string, SubscriptionProfile> sub_profiles = 6;
     string default_rule_id = 7;
+    // DEPRECATED
```

Comment:
This entire package is deprecated actually (there are no usages of it and there shouldn't be). Looks like I just forgot to clean this up with the configurator migration a long time ago. You can revert this change, I'll rm this package myself